packages/desktop/views/dashboard/Dashboard.svelte

```diff
@@ -110,6 +112,7 @@
         <div class="flex flex-col h-full dashboard-w">
             <svelte:component this={tabs[$dashboardRoute]} on:next={$appRouter.next} />
         </div>
+        <SideDrawer />
```

Comment:
Thinking it may be more "correct" to put the conditional rendering block here rather than in the component itself, i.e.
```svelte
{#if $sideDrawerState?.active}
    <SideDrawer />
{/if}
```